src/pnpm-lock.yaml

```diff
@@ -676,7 +676,7 @@ importers:
       string-width: ^4.2.0
       strip-ansi: 6.0.0
       strip-indent: 3.0.0
-      tar: ^6.0.1
+      tar: ^6.1.12
```

Comment:
I'm not entirely sure why, but we don't commit pnpm-lock updates, I think the release pipeline does that automatically. (Don't quote me on this)